ephemeral_storage_setup/utils.py

```diff
@@ -143,14 +143,15 @@ def populate_directory(directory, config):
     """
     Populate the given directory using specified config.
     """
+    method = config.get("method", "none")
```

Comment:
This could also be `None`. The conditionals below would continue to work without an additional `is not None` check. No big deal, though.

Reply:
Addressed in 65f73ba